app/femr/util/calculations/LocaleUnitConverter.java

```diff
@@ -18,6 +18,52 @@
      * @return VitalMultiMap with metric values
      */
     public static VitalMultiMap toMetric(VitalMultiMap vitalMap) {
+
+        for (int dateIndex = 0; dateIndex < vitalMap.getDateListChronological().size(); dateIndex++) {
+
+            // Get imperial temperature(F)
+            String tempC = vitalMap.get("temperature", vitalMap.getDate(dateIndex));
+
+            // If temp is not null convert to metric(C) and store in map as temperature
+            if (tempC != null) {
+                vitalMap.put("temperature", vitalMap.getDate(dateIndex), getCelcius(tempC));  // temperature is in Fahrenheit
+                vitalMap.put("temperatureC", vitalMap.getDate(dateIndex), tempC);   // added for femr-136 - dual unit display
```

Comment:
Is the C supposed to stand for Celsius or would this make more sense as "temperatureF" or "temperatureFahrenheit"?